schema/project-level/project-schema.json

```diff
@@ -62,6 +65,36 @@
       "description": "The period over which this project is planned to run. This may be updated during the preparation phase as information becomes available to more accurately specify anticipated start and end dates, but should not be updated during the implementation or completion phases. The planned completion date of the project should be provided in `period.endDate`, which can be compared with the actual completion date in `completion.endDate`",
       "$ref": "#/definitions/Period"
     },
+    "identificationPeriod": {
+      "title": "Identification period",
+      "description": "Identification refers to the decision to develop a project within the budget and programme of a project owner (the public entity responsible for executing the budget). This stage is also called definition.",
```

Comment:
I find it a bit odd that the codelist descriptions are reused here, and that there's no reference to "period".

I suggest using the format: "The period over which the {stage-name} stage is planned to run." (following the pattern set by `period`). The meaning of the stages is deferred to the projectStatus codelist.

Reply:
Sounds good - that'll be easier to maintain, too. Done in https://github.com/open-contracting/infrastructure/pull/432/commits/390d4b022836367b24bf997288e0d03e26a8e6c1